### DISCUSSION TITLE
Dart-like roadmap ?

### DISCUSSION THREAD

**Discussion by npenin** [microsoft/devicescript#553] [General]
Hi,

Just stubled upon this awesome project ! Any roadmap on supporting on the x86/64 ? That would potentially bring typescript to the level of Dart and give the best language ever (IMHO) a chance to compete with Dart, C# and others.

**Comment by pelikhan** [comment]
We do not have plans to target desktop applications; other than supporting a WASM simulator build. There is a number of specific constraints / design decision associated to small embedded MCUs and DeviceScript is primarily focused on this scenario (embedded).

>  give the best language ever (IMHO)

Can't argue with that :)
